x/market/keeper/keeper.go

```diff
@@ -276,11 +298,11 @@ func (k *Keeper) NewOrderSingle(ctx sdk.Context, aggressiveOrder types.Order) (*
 			nextSourceFilledCoin := sdk.NewCoin(passiveOrder.Source.Denom, stepSourceFilled.RoundInt())
 			err := k.transferTradedAmounts(ctx, nextDestinationFilledCoin, nextSourceFilledCoin, passiveOrder.Owner, aggressiveOrder.Owner)
 			if err != nil {
-				panic(err)
+				fmt.Println(nextDestinationFilledCoin, nextSourceFilledCoin)
```

Comment:
Use logging instead of `Println`. Error should probably be returned.

Reply:
:+1: yeah debugging slipped, i reverted to the original